apps/web/components/booking/BookingListItem.tsx

```diff
@@ -776,6 +824,29 @@ const BookingItemBadges = ({
       {booking?.assignmentReason.length > 0 && (
         <AssignmentReasonTooltip assignmentReason={booking.assignmentReason[0]} />
       )}
+      {booking.reports && booking.reports.length > 0 && (
+        <Tooltip
+          content={
+            <div className="space-y-1">
+              {booking.reports.map((report) => {
+                const reasonKey = `report_reason_${report.reason.toLowerCase()}`;
+                const reasonText = t(reasonKey);
+                const displayText = report.description ? `${reasonText}: ${report.description}` : reasonText;
+                return (
+                  <div key={report.id} className="text-xs">
+                    {displayText}
+                  </div>
+                );
+              })}
+            </div>
+          }>
+          <Badge className="ltr:mr-2 rtl:ml-2" variant="red">
+            {booking.reportedByCurrentUser
+              ? t("reported_by_you")
+              : t("reported_count", { count: booking.reports.length })}
+          </Badge>
```

Comment:
We don't need this. Once a booking is reported by any host, it should show up as reported for all other hosts as well. One way to see this is, a booking has a spam reported status, that can either be true or false, and once it is true, it is true for all hosts.